config/lint.xml

```diff
@@ -81,7 +81,7 @@
         <ignore path="**/WordPressEditor/**" />
         <ignore path="**/.gradle/caches/**" />
         <ignore path="**/generated/**" />
-        <ignore regexp=".*\/values\-([a-z]|[A-Z]|\-){2,6}\/strings\.xml$"/>
+        <ignore regexp=".*\/values\-([a-z]|[A-Z]|\-){2,6}\/.*\.xml$"/>
```

Comment:
I _think_ this is fine, but I would have preferred to be more verbose and add each file separately - assuming it's less than 5 or so files of course. My slight worry is that we'll miss something due to this catch all file name pattern. However, I can't find a concrete example where it'll be an issue, so I am just leaving this as a note.

Reply:
Good point, addressed in https://github.com/wordpress-mobile/WordPress-Android/pull/16293